_posts/2021-12-14-announcing-readonly.markdown

```diff
@@ -0,0 +1,94 @@
+---
+title: "Introducing `readonly`"
+layout: post
+author: jjwu
+category: blog
+---
+
+As of HHVM version 4.139, we've released a new keyword called `readonly` in Hack. `readonly` is a new feature in Hack for restricting the mutability of object properties. It's designed for use cases that require performant, runtime-enforced mutability control. 
+
+`readonly` has two main principles: **readonlyness* and **deepness*. 
```

Comment:
Is the mismatched numbers of `*` here intentional, i.e. do you want a literal `*` and italics?